lib/utils/utils.go

```diff
@@ -148,41 +148,14 @@ func KeystoreFilepaths(basepath string) ([]string, error) {
 	return keys, nil
 }
 
-// GetGssmrGenesisRawPathTest gets the gssmr raw genesis path
-// and fails the test if it cannot find it.
-func GetGssmrGenesisRawPathTest(t *testing.T) string {
-	t.Helper()
-	path, err := GetGssmrGenesisRawPath()
-	require.NoError(t, err)
-	return path
-}
-
-// GetGssmrGenesisRawPath gets the gssmr raw genesis path
+// GetGssmrGenesisRawPath gets the gssmr genesis path
```

Comment:
supernit can't we keep it to how it was?

```suggestion
// GetGssmrGenesisRawPath gets the gssmr raw genesis path
```